Packs/EWS/Integrations/EWSO365/EWSO365.py

```diff
@@ -2215,15 +2220,17 @@ def fetch_last_emails(
     qs = qs.filter().order_by("datetime_received")
 
     result = []
+    exclude_ids = exclude_ids if exclude_ids else set()
```

Comment:
```suggestion
```
already handled in `fetch_emails_as_incidents` function

Reply:
But fetch_last_email does not have to get it. Think it is better to keep it for testing of the specific function.

Reply:
then add it also to EWSv2

Reply:
I did :)